keyboards/nix_studio/n60a/keymaps/via/n60a_via.json

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards)

Also, you may have noticed that "via.json" files are not included by default, and you probably had to rename the file to get it to be added.  This is intentional, for exactly this reason.